src/incubator/Dialog/helpers/useSizeStyle.tsx

```diff
@@ -0,0 +1,65 @@
+import {isUndefined} from 'lodash';
```

Comment:
Is this hook meant to convert string percentages to numbers? 
Cause react-native already supports string percentages

Reply:
This is what I've talked with you about; RN does have this functionality, but it does not work because of the different views in the Dialog, I did not find a way to not break other things and have this work 😞

Reply:
Let's go over it together maybe we can find a solution